packages/@webex/plugin-cc/src/services/task/index.ts

```diff
@@ -86,5 +86,92 @@ export default class Task implements ITask {
     }
   }
 
-  // TODO: Hold/resume, recording pause/resume, consult and transfer public methods to be implemented here
+  /**
+   * This is used to hold the task.
```

Comment:
Have added comments, documentation and methods for each call control. Note that the promise accept/reject is handled in aqm-reqs and we supply the events in the contacts.ts of the task folder (as done in Priya's PR). Everything works smoothly.